### DISCUSSION TITLE
I can't get the student develop pack.

### DISCUSSION THREAD

**Discussion by leoch627** [community/community#52348] [GitHub Education]
<img width="226" alt="image" src="https://user-images.githubusercontent.com/67500557/230750797-9fec3f15-6ea4-46cd-9c2b-9e830deb96fe.png">
I don't have a email address from my school. What should I do?

**Comment by leoch627** [comment]
Any ideas?

**Comment by samus-aran** [comment]
Hi @leochang815 👋 

Here is a quick tip on submitting evidence:

- If your original document is not in English then we advise you to take a photo of the original next to an English translation. This also include your profile information.
- Ensure your evidence correspond to details in your GitHub profile. The verification process looks at your GitHub username, email and name ( as well as other points) to cross reference the evidence submitted. Do you use a variation of your name or a nickname?  It’s these small but important points to consider when applying that can make a much smoother process.
- We would also suggest that when you update your profile information that you log out and log back into GitHub before re-applying.
- Ensure you select the most appropriate **evidence type** in the dropdown. The order of options indicates the preference of evidence with Academic Email being at the top. If you don't have an Academic email or Student ID, do try and acquire evidence that is positioned higher rather than lower in that option list.

If you have been repetitively unsuccessful using a particular evidence type (e.g. Student ID) we would recommend that you acquire and submit an alternative piece of evidence that can verify your student status.

Let us know how you get on 👍
